downstream/modules/platform/ref-controller-license-support.adoc

```diff
@@ -0,0 +1,10 @@
+[id="ref-controller-license-support"]
+
+= Support
+Red Hat offers support to paid {PlatformName} customers.
+
+If you or your company has purchased a subscription for {PlatformNameShort}, you can contact the support team at https://access.redhat.com 
```

Comment:
```suggestion
If you or your company has purchased a subscription for {PlatformNameShort}, you can contact the support team at https://access.redhat.com.
```

Reply:
Tried adding a full stop here, it becomes part of the URL and hence might fail.